CMakeLists.txt

```diff
@@ -57,7 +56,7 @@ if(CMAKE_INSTALL_PREFIX STREQUAL "${AliPhysics_SOURCE_DIR}")
 endif()
 
 # Path to additonal modules
-set(CMAKE_MODULE_PATH "${AliPhysics_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH "${AliPhysics_SOURCE_DIR}/cmake" "${AliPhysics_SOURCE_DIR}/OADB")
```

Comment:
Hi, @dberzano,
thank you for your information! 
However, I am instructed to add a line '${AliPhysics_SOURCE_DIR}/OADB/' to the list of 'additional includes' (the list starts at line 23 of the CMakeLists.txt file), in order that I could use the AliEventCuts class. So, where should I put the line '${AliPhysics_SOURCE_DIR}/OADB/' in the CMakeLists.txt?
Thanks,
Au-Au